apps/cli/lib/mix/tasks/mana.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Mix.Tasks.Mana do
+  @moduledoc """
+  Main entry-point for the mana CLI.
+
+  Command Line Options:
+
+      `--chain` - Chain to load data from (default: ropsten)
```

Comment:
Is the default `ropsten` for the time being, or is this the idea as we go forward? Would mainnet be a better default?

Reply:
Yes, I didn't actually know what chain to use for mainnet, so that's actually why. But I think as we get closer to launch, we can start to clean up these types of defaults to more sensible values.